filebeat/reader/encode/encoding/utf16.go

```diff
@@ -29,7 +29,7 @@ import (
 type endianness int8
 
 const (
-	unknownEndianess endianness = iota
+	unknownEndianness endianness = iota
```

Comment:
This is odd. I can't find it outside this project, so I'm hoping I can make this change.